CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# How To Contribute
+There are several ways to contribute to the development of ReactOS. The most often encountered problem is not knowing where to begin or what to do. If you are able to program or understand the technical information that is pertinent to this project, helping the development can be easy.
+
+## Documentation
+
+There are some important points if you'd like to help document ReactOS:
```

Comment:
In written texts, it is always better not to use abbreviated forms. So:
you'd --> you would
doesn't --> does not
etc... (there are others in the text),

Reply:
FYI this was copied from the Wiki (most of the text).